src/colornames.csv

```diff
@@ -26644,6 +26726,7 @@ Teewurst,#f2dbd7,
 Tegreen,#c1b65f,
 Teldrassil Purple,#ad66d2,
 Telemagenta,#aa22cc,x
+Telfor Tower,#595d56,

```

Comment:
I think this is a typo in the Original name, found something called "Telford Tower". Lets remove it: its not like its clear what color this tower is anyway.